code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -783,40 +751,41 @@
 	if(src.controller)
 		src.release_control()
 	src.flock?.removeDrone(src)
-	// create the flockbits
-	animate_flock_drone_split(src)
-	var/mob/living/critter/flock/bit/B
-	// get candidate places to move them
```

Comment:
useful comment

Reply:
Thinking about this some more, think it should be using is_flock_blocked_turf(), not is_blocked_turf() for bits?

Reply:
I'm fine with that, there's probably an instance or two where that needs changing